src/app/pages/security/security.page.ts

```diff
@@ -19,8 +21,11 @@ export class SecurityPage implements OnInit {
     public color: string;
     public securityService: SecurityService;
 
-    constructor(public platform: Platform, private dialogs: Dialogs) {
-        this.securityService = new SecurityService();
+    constructor(public platform: Platform,
+                private dialogs: Dialogs,
+                private openShift: OpenShiftService) {
+        const configurationService = openShift.getConfig();
+        this.securityService = new SecurityService(new MetricsService({ configuration: configurationService.getConfigByType('metrics') }));
```

Comment:
Let's create a method to get metrics config instead of getting config by type. I was trying to build service that can be reusable by our community

Reply:
I don't like the idea of adding a `get[ServiceType]Configuration` for each service we have or we will create in the future: I think that would clutter our interface.
I added an enumeration with all our services and a `getConfiguration` method that receives that enum as param